content/nldi.md

```diff
@@ -54,43 +55,59 @@ The public face of the NLDI is a search service that takes a watershed identifie
 How is the NLDI service structured?
 -----------------------------------------------------------------------
 
-See the swagger documentation here: [https://labs.waterdata.usgs.gov/api/nldi/swagger-ui.html](https://labs.waterdata.usgs.gov/api/nldi/swagger-ui.html)
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data` returns indexed feature sources.
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data/featureSource/featureID/navigation` returns the navigation options for that feature ID from the given feature source.
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data/featureSource/featureID/navigation/mode/dataSource` returns the data derived from a navigation with the provided mode for the requested data source.
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data/featureSource/featureID/basin` returns a basin boundary upstream of that featureID.
```

Comment:
I'd recommend adding an indicator as to which parts of the url are variables. (e.g. `https://labs.waterdata.usgs.gov/api/nldi/linked-data/{featureSource}/{featureID}/navigation`)

Reply:
Just added. 
![image](https://user-images.githubusercontent.com/1492803/163579578-b67dc3c1-f0c2-4e8c-9af0-95b61c0b9ed3.png)